### DISCUSSION TITLE
Have the timer also listen for mouse events while in desktop keyboard mode

### DISCUSSION THREAD

**Discussion by alexkatz** [kash/cubedesk#153] [Ideas]
Having the ability to use my trackpad to start and stop the timer would be super helpful, as sometimes it's easier than reaching for spacebar. 

This especially goes for folks like myself who have custom/split keyboards without a prominent spacebar, but do have a trackpad sitting on their desk!

Regardless, phenomenal app. I'm a happy pro member for the year as I'm working on (ever so slowly) improving my Roux.

**Comment by JupiLogy** [comment]
> split keyboards

based

My hardware solution for this is to have a custom layer where like half the buttons on the entire keyboard are spacebar.

Just thought I'd throw in this suggestion, in case this doesn't get added for a while.

**Reply by JupiLogy** [reply]
https://configure.zsa.io/moonlander/layouts/DaO0b/latest/4

example ^

**Reply by alexkatz** [reply]
Nice. That's a great idea. Maybe I'll try it. The idea of smashing my hand aimlessly into my keyboard after every solve is pretty hilarious.
